test/storage/storage_util_test.cpp

```diff
@@ -198,4 +197,45 @@ TEST_F(StorageUtilTests, ApplyDelta) {
   }
 }
 
+// Verifies that we properly generate both a valid BlockLayout and a valid mapping from col_oid to col_id for a given
+// random Schema
+// NOLINTNEXTLINE
+TEST_F(StorageUtilTests, BlockLayoutFromSchema) {
```

Comment:
Same here. It looks like it's small, but I think we should make it its own test file under `catalog`

Reply:
I don't agree with this one. The function being tested is BlockLayoutFromSchema in storage_util

Reply:
Yeah I was thinking we move that one too. Low priority, can always get that later.